UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1063,18 +1076,29 @@ PlatformBootManagerAfterConsole (
   BOOLEAN                        NetBootEnabled;
   BOOLEAN                        BootMenuEnable;
   UINTN                          VarSize;
+  EFI_EVENT                      Event;
+  EFI_INPUT_KEY                  Enter;
 
   Black.Blue = Black.Green = Black.Red = Black.Reserved = 0;
   White.Blue = White.Green = White.Red = White.Reserved = 0xFF;
 
-  EfiBootManagerConnectAll ();
+  gST->ConOut->EnableCursor (gST->ConOut, FALSE);
   gST->ConOut->ClearScreen (gST->ConOut);
+
   WarnIfRecoveryBoot ();
 
   BootLogoEnableLogo ();
 
+  //
+  // Register ENTER as CONTINUE key
+  //
+  Enter.ScanCode    = SCAN_NULL;
+  Enter.UnicodeChar = CHAR_CARRIAGE_RETURN;
+  EfiBootManagerRegisterContinueKeyOption (0, &Enter, NULL);

```

Comment:
Does this include Enter on numpad? I always find it annoying when it doesn't work.

Reply:
Well, haven't checked :) But probably yes.

Reply:
Yup, NUM ENTER also works (at least with USB KBD)